core-services/prow/02_config/_plugins.yaml

```diff
@@ -3491,6 +3506,32 @@ plugins:
   - wip
   - yuks
   - approve
+  integr8ly/grafana-operator:
+  - assign
+  - blunderbuss
```

Comment:
We removed this plugin (blunderbuss) for other integr8ly repos, up to you if you want to keep it though.

Reply:
I'll remove it here as well, we'd probably end up ignoring the blunderbuss requests anyway since it's either me or Peter reviewing